src/main/java/view/LoginView.java

```diff
@@ -132,6 +132,7 @@ public void propertyChange(PropertyChangeEvent evt) {
 
     private void setFields(LoginState state) {
         usernameInputField.setText(state.getUsername());
+        passwordInputField.setText(state.getPassword());
```

Comment:
This does follow proper usage of the helper function to update the state's text fields.